OpenDreamClient/Interface/DreamInterfaceManager.cs

```diff
@@ -270,8 +271,10 @@ private void RxLink(MsgLink message) {
         // TODO: This can be a topic call or a connection to another server
         if (uri.Scheme is "http" or "https") {
             _uriOpener.OpenUri(message.Url);
+        } else if (uri.Scheme is "ss14" or "ss14s") {
```

Comment:
You can check `_gameController.LaunchState.FromLauncher` to display a message if it can't redial.